### DISCUSSION TITLE
Welcome to IdleonCompanion Discussions!

### DISCUSSION THREAD

**Discussion by Musimaniac** [Idleon-Companion/IdleonCompanion#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dougllima** [comment]
Hi guys! Awesome work here!
First of all, I'm a web developer for almost 10 years, but never was active on any open-source project.
One week ago i started a little project to do a tracker to idleon, using something like [this extension](https://github.com/Corbeno/Idleon-Api-Downloader)
And them i found your site, and this awesome community.
So, can i join you guys on the code stuff 😁?
